src/index.js

```diff
@@ -274,8 +265,7 @@ class NearProvider {
                 result
             });
         }, (err) => {
-            console.error(err);
-            throw new Error(`NearProvider: ${err}`);
+            cb(err)
```

Comment:
fix here --^ though `send` never gets called in our tests

Reply:
oh we should call that haha.